README.md

```diff
@@ -24,6 +32,23 @@ This library requires:
 1. Python 3.8 or higher; and
 2. Linux, Windows, or macOS operating system.
 
+## Versioning
```

Comment:
Moved up for visibility. My expectation is for most folk to stop reading when they get to the getting started & info on job attachments and the like. The information on versioning is super important, since it tells the end-user what they can expect w.r.t. interface stability as it relates to version numbers.